### DISCUSSION TITLE
Is Non-Integer Multiple-Cell-Height Design supported?

### DISCUSSION THREAD

**Discussion by inv1s10n** [The-OpenROAD-Project/OpenROAD#5718] [Q&A] [answered]
I want to know if Non-IntegerMultiple-Cell-Height Design(or hybrid row height design) is supported by openROAD. Such as finflex in TSMC.

**Comment by maliberty** [comment]
It was implemented but has some issues.  I'm in the process of fixing it up though it is going somewhat slowly.  We do intend to support such.

**Reply by inv1s10n** [reply]
Do cases of openROAD have such a design now?

**Reply by maliberty** [reply, marked as answer]
For testing https://github.com/The-OpenROAD-Project-private/OpenROAD-flow-scripts/tree/master/flow/designs/gf180/aes-hybrid

**Reply by inv1s10n** [reply]
hello，can i run this case in openroad? (not openroad flow  script)

**Reply by maliberty** [reply]
You would have to convert it by hand - why not use ORFS?

**Reply by inv1s10n** [reply]
Just because I used OpenRoad before, but now I'm using ORFS. Thank you

**Reply by inv1s10n** [reply]
> For testing https://github.com/The-OpenROAD-Project-private/OpenROAD-flow-scripts/tree/master/flow/designs/gf180/aes-hybrid

I have another question. How does OpenRoad generate this hybrid row layout? Is it automatically generated by Floorplan or manually modified?

**Reply by maliberty** [reply]
It is automatic from a site with a ROW_PATTERN

**Reply by inv1s10n** [reply]
Really sad. I do not find this key word 
![image](https://github.com/user-attachments/assets/c3371b68-c1d4-4aea-82fe-4e2beee8cc09)

**Reply by maliberty** [reply]
https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/a8f7b9644a89a2867c507be22acc35de54abd322/flow/designs/gf180/aes-hybrid/hybrid_sites.lef#L27
